src/winforms/toga_winforms/command.py

```diff
@@ -3,6 +3,7 @@ class Command:
     def __init__(self, interface):
         self.interface = interface
         self.native = []
+        self.set_enabled(None)
```

Comment:
Why None? Wouldn't False make more sense here?

Reply:
Actually... why is this needed at all? When the command is created, it should be setting `self.enabled`, which should be calling `set_enabled()` with the initial value. Is there an edge case that requires this (and if so - we should document that inline with a comment)